insights/specs/datasources/ssl_certificate.py

```diff
@@ -134,3 +134,39 @@ def rsyslog_tls_cert_file(broker):
                         if 'DefaultNetstreamDriverCertFile' in part:
                             return part.split('=')[-1].strip('"')
     raise SkipComponent
+
+
+@datasource(RsyslogAllConf, HostContext)
+def rsyslog_tls_ca_cert_file(broker):
+    """
+    Get the rsyslog tls ca certificate file path configured by "DefaultNetstreamDriverCAFile"
+
+    Arguments:
+        broker: the broker object for the current session
+    Returns:
+        str: Returns the SSL certificate file path configured by "DefaultNetstreamDriverCAFile"
+    Raises:
+        SkipComponent: Raised if "DefaultNetstreamDriverCAFile" isn't found
+    """
+    rsyslog_objs = broker[RsyslogAllConf]
+    for obj in rsyslog_objs.values():
+        for item in obj:
+            if 'DefaultNetstreamDriverCAFile' in item:
+                if '$DefaultNetstreamDriverCAFile' in item:
+                    # basic format
+                    return item.split()[-1].strip()
+                else:
+                    # advanced format
+                    # it is set in global block, and the global line contains all the content in it
+                    parts = item.split()
+                    for part in parts:
+                        if 'DefaultNetstreamDriverCAFile' in part:
```

Comment:
And seems we can add a `break` to this `if`'s else case, to save some `parts` iteration try.

Reply:
@JoySnow updated based on the two comments, please help to review again, thanks.